documentation/modules/about-cold-warm-migration.adoc

```diff
@@ -20,6 +20,15 @@
 
 Cold migration is the default migration type. The source virtual machines are shut down while the data is copied.
 
+[NOTE]
+====
+VMware only: In cold migrations, in situations in which a package manager cannot be used during the migration, {project-short} does not install the daemon `qemu-guest-agent` on the migrated VMs. This has some impact on the functionality of the migrated VMs, but overall, they are still expected to function.
```

Comment:
```suggestion
For VMware: In scenarios where you cannot use a package manager during a cold migration, {project-short} does not install the `qemu-guest-agent` daemon on the migrated VMs. This impacts the functionality of the migrated VMs, but overall, the VMs function as expected.
```

Reply:
```suggestion
VMware only: In cold migrations, in situations in which a package manager cannot be used during the migration, {project-short} do not install the daemon `qemu-guest-agent` on the migrated VMs. This has some impact on the functionality of the migrated VMs, but overall, they are still expected to function.
```

Reply:
No -- MTV "does not".

Reply:
@RichardHoch, my bad, ignore that comment.
Is it possible to not add that information in the admonition considering the module has only one line in the abstract? 
You can use the definition list for VMware and Fallback options with a description added before it to describe the relation between these points and cold migration.

Reply:
@apurvabhide17 I don't understand your suggestion. {project-short} is rendered as MTV in this repo. Would you write the note differently? The note only applies to cold migrations in which the user is migrating VMs from VMware to CNV (all MTV migrations are to CNV).

Reply:
@RichardHoch, added my suggestion related to this where the note starts.

Reply:
@RichardHoch @shdeshpa07, how about replacing "In cold migrations, in situations in which a package manager cannot be used during the migration," with "In cold migrations, where a package manager cannot be used during the migration, [..] to avoid fluff?